packages/client/modules/userDashboard/components/OrgBilling/OrgBillingCreditCardInfo.tsx

```diff
@@ -83,13 +83,29 @@ const OrgBillingCreditCardInfo = (props: Props) => {
     `,
     organizationRef
   )
-  const {creditCard, id: orgId, orgUserCount} = organization
-  const {modalPortal, closePortal, togglePortal} = useModal()
+  const {creditCard} = organization
+  const [isUpdating, setIsUpdating] = useState(false)
   if (!creditCard) return null
-  const {activeUserCount} = orgUserCount
   const {brand, last4, expiry} = creditCard
+
+  if (isUpdating) {
+    return (
+      <StyledPanel label='Credit Card'>
+        <StyledRow>
+          <Elements stripe={stripePromise}>
+            <UpdatePayment setIsUpdating={setIsUpdating} />
```

Comment:
+1 would be nice to indicate this is just temporary

Reply:
🤔 why would it be temporary?

Reply:
It's not updating anything yet, is it? I don't see any action wired up for 'submit'

Reply:
When the user clicks "Cancel", it updates the `isUpdating` state to false so that the update payment UI closes. I've created a `handleCancel` function and passed that here so the name changes. In the following PR, I add logic that handle the "submit" case.